src/Avalonia.Controls/ProgressBar.cs

```diff
@@ -100,12 +100,18 @@ public double Container2AnimationEndPosition
         private double _indeterminateEndingOffset;
         private Border? _indicator;
 
+        public static readonly StyledProperty<double> PercentageProperty =
+            AvaloniaProperty.Register<ProgressBar, double>(nameof(Percentage));
```

Comment:
It should be readonly DirectProperty. And `Percentage` should have private setter.

Reply:
As it's not expected to set this property outside of this control